src/pages/event.tsx

```diff
@@ -0,0 +1,157 @@
+import { useLocale } from "@/locales";
+import { useState, useRef } from "react";
+function SingleEvent({
+  bgColor,
+  borderColor,
+  toReverse,
+  textColor,
+  image,
+  eventInfo,
+  price,
+}: {
+  bgColor: string;
+  borderColor: string;
+  toReverse: boolean;
```

Comment:
This property isn't passed in at all call sites. Either pass it in as false where it isn't true or make this line `toReverse?: boolean;` so that it's optional

Reply:
Fixed, this property is now passed in at all call sites.